Vision/proposal.md

```diff
@@ -0,0 +1,62 @@
+# The Mission of the W3C
+
+The W3C leads the community in defining a World Wide Web that is inclusive and respectful of its users, 
+to empower an ethical, equitable, informed, and interconnected society.
+
+## **Introduction**
+### The World Wide Web
+
+The World Wide Web was originally conceived as a tool for sharing information. 
+It has evolved rapidly into a fundamental part of humanity, 
+sparking major social change by providing access to 
+knowledge, education, commerce and shopping, social experiences, civic functions, entertainment, and more.
+
+The Web's amazing success has also led to many unintended consequences that harm society: 
+openness and anonymity have given rise to scams, phishing, and fraud. 
+The ease of gathering personal information has led to business models that mine and sell detailed user data, without people's awareness or consent. 
+Rapid global information sharing has allowed misinformation to flourish and be exploited for political and commercial gain. 
+Technology is not neutral; 
+new technologies enable new actions and new possibilities, and we must take responsibility to address the impacts of the Web.
+
+We believe the World Wide Web should be inclusive and respectful of its users: 
+a Web that supports truth over falsehood, people over profits, humanity over hate. 
```

Comment:
I'm a bit uncomfortable with L22 phrasing, e.g. "truth over falsehood" kind of feels "we" would define what truth is and makes sure the Web censors whatever doesn't fit. Society (or societies, really) has developed plenty of mechanisms to determine what it collectively wants to support, incl. in terms of avoiding harms that would derive from falsehoods. I think we should strive to make the Web support the implementation of these mechanisms, and avoid undermining them - not necessarily encode what's their outcome should be. I'm not sure how to formulate that in a way that doesn't feel administrative.

Reply:
filed #113